packages/devextreme/scss/widgets/fluent/tabPanel/_index.scss

```diff
@@ -11,7 +11,6 @@
 
 .dx-tabpanel {
   .dx-tabpanel-tab {
-    min-width: 72px;
```

Comment:
let's add a test to check our real scenario?
Actual screenshots do not show it

Reply:
New tests have been added there [b778453472fb833bf272bb65b9c98be08e113769].